src/core/Akka.TestKit/TestKitBase.cs

```diff
@@ -630,7 +673,7 @@ public IActorRef ChildActorOf(Props props, string name, CancellationToken cancel
         /// <returns></returns>
         public IActorRef ChildActorOf(Props props, CancellationToken cancellationToken = default)
             => ChildActorOfAsync(props, cancellationToken)
-                .ConfigureAwait(false).GetAwaiter().GetResult();
+                .Result;
```

Comment:
We still need to use GetAwaiter().GetResult() because this automatically unwraps all AggregateException, using .Result here breaks behavior consistency.